README.md

```diff
@@ -0,0 +1,45 @@
+# To process a dataset:
+
+## Setup
+### Expected files in a dataset directory:
+- `dataset.json`: a json file with metadata about the dataset and the names of the other files, using the keys listed below
+- featureDefs: a json describing the measured features in this dataset, *the order of this document will dictate the display order of the features on the website.*
```

Comment:
My understanding is that the display order is supposed to be defined in dataset.json, but if none is defined then the order of featureDefs should be used.  

The ordering of featureDefs gives the order for the features in the arrays in featuresData.

Reply:
yeah, you're right, I'll make this clearer.